src/bridge/IrcHandler.ts

```diff
@@ -956,6 +955,13 @@ export class IrcHandler {
         return metrics;
     }
 
+    public onConfigChanged(config: IrcHandlerConfig) {
+        const newMentionMode = config.mapIrcMentionsToMatrix || "on";
+        if (this.mentionMode !== newMentionMode) {
```

Comment:
Is there any point in creating a new const and comparing? Cheaper to just do `this.mentionMode = config.mapIrcMentionsToMatrix || "on";` ?